app/(tabs)/_layout.tsx

```diff
@@ -55,7 +55,7 @@ export default function TabLayout() {
           headerShown: false,
           title: strings[language].tabs.preview,
           tabBarIcon: ({ color }) => <TabBarIcon name="eye" color={color} />,
-          tabBarTestID: 'preview-tab',
+          tabBarButtonTestID: 'preview-tab',
```

Comment:
refreshing memory - this is a new update right? probably warrants a doc update here https://github.com/appcues/appcues-react-native-module/blob/main/docs/AnchoredTooltips.md#bottom-tab-navigation

Reply:
Right, this was a change in `@react-navigation/native@7`. I'll update that doc :+1: